packages/carbon-react/src/components/Icons/Icons.stories.js

```diff
@@ -9,6 +9,7 @@ import './Icons.stories.scss';
 import React from 'react';
 import { Bee, Bicycle, ChevronUp } from '../../../icons';
 
+// eslint-disable-next-line storybook/csf-component
```

Comment:
Why do these lines need to be disabled?

Reply:
@abbeyhrt basically the rule is that every story _must_ have a `component:` key, which is great except we sometimes have stories that aren't for a specific component. This rule therefore doesn't really apply and so we ignore it 👀